01 Linear Regression (complete).ipynb

```diff
@@ -55,18 +55,18 @@
        "  <tbody>\n",
```

Comment:
<p>Check out the new visualizations</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jamesdanielwhitford/linear_regression_example/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>